src/models/Card.js

```diff
@@ -97,7 +103,15 @@ function save(next) {
   return next();
 }
 
+function setIsValid() {
+  return !cardValidationByTemplate(this.template)
+    .validate(this, { allowUnknown: true }).error;
```

Comment:
pour plus de lisibilité je mettrais `cardValidationByTemplate(this.template).validate(this, { allowUnknown: true })` dans une variable

Reply:
En soit je mets le résultat directement dans isValid, je ne vois pas comment bien nommer cette variable. As tu des idées ?